spec/SEOAssessorSpec.js

```diff
@@ -32,4 +32,16 @@ describe ( "running assessments in the assessor", function() {
 		assessor.assess( new Paper( "text", { url: "sample url" } ) );
 		expect( assessor.getValidResults().length ).toBe( 7 );
 	});
+
```

Comment:
Unrelated to this PR but since we're here... seems to me in this file at line 3 this:
`var AssessmentResult = require( "../js/values/AssessmentResult.js" );`
is not used anywhere and could be removed?